packages/syntax-floating-components/src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  MutableRefObject,
+  ReactElement,
+  ReactNode,
+  useLayoutEffect,
+  useRef,
+  useMemo,
+} from "react";
+import classnames from "classnames";
+import {
+  Placement,
+  arrow,
+  flip,
+  useFloating,
+  shift,
+  offset,
+} from "@floating-ui/react";
+
+import styles from "./Tooltip.module.css";
+
+const defaultPlacement = "top";
+
+const Tooltip = ({
+  children,
+  offset: offsetValue = 7, // include arrow size which equals sqrt(5^2 + 5^2) = 7.07 ~ 7,
```

Comment:
What does offset do and what type of value is the number used for? Is it pixels?

Reply:
It creates a gap b/w the floating element from the anchor element. Yes it's in pixels.

https://floating-ui.com/docs/offset